notebooks/examples/create_a_custom_suite.ipynb

```diff
@@ -0,0 +1,759 @@
+{
```

Comment:
<p>I think that it's better to put the imports with the code for copy &amp; paste simplicity.</p><p>also I'd add after the definition a print of the suite</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>